testrunner/execute_test.go

```diff
@@ -86,32 +60,6 @@ func TestRunTests_missingFunc(t *testing.T) {
 			t.Errorf("Missing function test run - unexpected suffix in line: %s, want: %s", lines[i], expectedSuffix)
 		}
 	}
-
-	output := &testReport{
-		Status:  statErr,
-		Version: 2,
-		Message: res,
-	}
-	btr, err := json.MarshalIndent(output, "", "\t")
-	if err != nil {
-		t.Errorf("Missing function test output not valid json: %s", err)
-	}
-	tr := string(btr)
-
-	pre := `{
-	"status": "error",
-	"version": 2,
-	"message": "# github.com/exercism/go-test-runner/testrunner/testdata/practice/missing_func`
-
-	post := `returned exit code 2: exit status 2",
-	"tests": null
-}`
-	if !strings.HasPrefix(tr, pre) {
-		t.Errorf("Missing function test run unexpected json prefix: %s", tr)
-	}
-	if !strings.HasSuffix(tr, post) {
-		t.Errorf("Missing function test run unexpected json suffix: %s", tr)
-	}
```

Comment:
In an earlier PR I added the code that checks `res` line by line (L74-87). Since `res` is now already checked, the code I removed here was only testing json.MarshalIndent works on a struct that was set up here in the test case. That is unnecessary as part of this test case, same for the other tests where I removed this code. There are other test cases to cover the Marshaling process so we still ensure that works as expected.